docs/modules/fs_mitm.md

```diff
@@ -1,2 +1,2 @@
 # fs_mitm
-fs_mitm is a sysmodule that enables intercepting file system operations. This module can log, deny, delay, replace, or redirect any request made to the filesystem. It enables LayeredFS to function, which allows for game mods.
+fs_mitm is a sysmodule that enables intercepting file system operations. This module can log, deny, delay, replace, or redirect any request made to the filesystem. It enables LayeredFS to function, which allows for replacment of game assets.
```

Comment:
replacEment

Reply:
should probably check my spelling before I push docs huh